types/chartist/index.d.ts

```diff
@@ -99,7 +99,7 @@ declare namespace Chartist {
     interface IChartistSeriesData {
         name?: string;
         value?: number;
-        data?: Array<number>;
+        data?: Array<number> | Array<{ x: number, y: number }> | Array<{ x: Date, y: number }>;
```

Comment:
I'd say that we could probably simplify this down to 

```ts
data?: Array<number> | Array<{ x: number | Date, y: number}>;
```

It might make it easier to figure out when the compiler sees you made a mistake and does that thing where it says "Can not cast {x} to {x} | {y} | {z} | null | undefined | {}"

Reply:
Agreed, i've just refacto as you suggested.

Reply:
Cool. 👍

Reply:
Thanks Matt for this quick and pleasant review, 
I would propose a test case later, but for now i work on urgent project using timestamp on x-axis and must hurry up to deliver it in time.
Do you think the build will be done quickly on npm side "@types/chartist": "^0.9.45" ?

Reply:
Unit test added